libs/sdk-flutter/lib/native_toolkit.dart

```diff
@@ -16,8 +17,11 @@ BreezSdkCore getNativeToolkit() {
       // On Linux the lib needs to be in LD_LIBRARY_PATH or working directory
       _breezSDK = BreezSdkCoreImpl(DynamicLibrary.open(_libName));
     } else if (Platform.isIOS || Platform.isMacOS) {
-      // iOS and macOS are statically linked
-      _breezSDK = BreezSdkCoreImpl(DynamicLibrary.process());
+      try {
+        _breezSDK = BreezSdkCoreImpl(DynamicLibrary.open("$_iosLibName.framework/$_iosLibName"));
```

Comment:
I may have overlooked something but this does not look right to me. Where does `"breez_sdk.framework/breez_sdk"` point to?

Reply:
For production we moved to dynamic loading where the pod download the framework on install.
iOS then expand this dynamic linked framework like this.